floodviz/tests/test_utils.py

```diff
@@ -0,0 +1,210 @@
+import unittest
+import requests_mock
+from floodviz.utils import parse_rdb
+
+
+class TestRdbParsePeak(unittest.TestCase):
```

Comment:
Please look at the previous comments for tests. Please follow the convention of the test class being TestRdbParse (TestCamelCaseFunctionBeingTested). You should only need a single TestCase class. Also please include a test for a response where no data lines are returned.

As I said before, it is not necessary to exactly mimic a call that you are planning to make nor mimic a response that you will get. The test will be easier to read, if you mock a response that has a few lines of comments, just a few columns and three or four rows of data.